### PR TITLE
MetricService added 

### PR DESCRIPTION
**Pull request by Kutaiba-Nezar-Kashmar**
- Create new service called Metrics
- Implemented logic to fetch a persons movie contribution both as a cast or a crew member
- made logic that calculate the following:
    -  CalculateNumberOfMovies
    - CalculateAverageMovieRatingAsCast
    - CalculateAverageMovieRatingAsCrew
    - CalculateKnownForGenre
- Made an endpoint so the front end can get statistics
- Unit test to test the calculation logic

### CODE REVIEW

File: src/Services/Metrics/Metrics.Test/UnitTest1.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    public void Setup()
+    {
+    }
+
+    [Test]
+    public void Test1()
+    {
+        Assert.Pass();
+    }
+}
```
Comment (michaelbui99) [inline src/Services/Metrics/Metrics.Test/UnitTest1.cs:15]: Can remove this

File: src/Services/Metrics/Metrics.API/Program.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+using Metrics.Application;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
```
Comment (michaelbui99) [inline src/Services/Metrics/Metrics.API/Program.cs:11]: I think these are middlewares in MovieInformation, but this is fine as well. No need to change

**Review by michaelbui99 — CHANGES_REQUESTED** [review]
noice wok

File: src/Services/Metrics/Metrics.Test/UnitTest1.cs (hunk 1 above)
Comment (Kutaiba-Nezar-Kashmar) [reply src/Services/Metrics/Metrics.Test/UnitTest1.cs:15]: done 👍

**Review by michaelbui99 — APPROVED** [review]
LGTM!

**Pull request merged**